### DISCUSSION TITLE
Photo on Page<title>

### DISCUSSION THREAD

**Discussion by Trout48** [community/community#41517] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, 
I have tried to upload a photo to my GitHub page and when I launch the page everything works great besides my photo is not uploading to my web page. I followed the instruction and even reloaded the page. I used My project.png for my photo in my repository.

**Comment by airtower-luna** [comment]
Please include a link to the repository where you have the problem, otherwise it's hard to help. I see you have at least two repositories with pages sites. It's best if you link the line in the HTML where you include the picture.

**Reply by Trout48** [reply]
[https://trout48.github.io/My-Resume.github.io/](url)

`<nav class="navbar navbar-expand-lg navbar-dark bg-primary fixed-top" id="sideNav">
            <a class="navbar-brand js-scroll-trigger" href="#page-top">
                <span class="d-block d-lg-none">Reece Heinrich</span>
                <span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="/assets/My project.PNG" alt="..." /></span>
            </a>`
I believe that I link them right in the repo. For the photo, I used PNG because that was recommended to me. 


Thank you for your reply!

**Comment by radar-blip** [comment]
Probably because of the space in your `My project.png` (in the index.html also).
Try removing the space and see what happens.

**Reply by Trout48** [reply]
Thank you for your reply. I have tried what you mentioned and it didn't work.

**Reply by radar-blip** [reply]
Your image in the assets folder still has a space in it. It must match the link.
Your link is like this: `<span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="/assets/Myproject.PNG" alt="..." /></span>`

So, the image should be named accordingly, without the space also, like so:  Myproject.PNG

**Reply by Trout48** [reply]
I can't believe that I missed that, thank you. I have changed it but still having the issue even when I change .PNG to .png.

**Comment by airtower-luna** [comment, marked as answer]
I see two problems in your current code:

```html
<span class="d-none d-lg-block"><img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="/assets/Myproject.png" alt="Trout48" /></span>
```

1. URLs are case sensitive, the spelling must be *exactly* the same. The file in your repository is called `Myproject.PNG`, not `Myproject.png`
2. You're using an absolute path, starting with a slash. That means the browser will request `https://trout48.github.io/assets/Myproject.png`, which wouldn't exist even with the capitalization fixed. You should use a relative path, that means relative to the directory where the current file (`index.html`) is.

So this should work:
```html
<img class="img-fluid img-profile rounded-circle mx-auto mb-2" src="assets/Myproject.PNG" alt="Trout48" />
```

**Reply by Trout48** [reply]
Thank you so much it worked. The GitHub community is awesome.
I wish you well!

**Reply by airtower-luna** [reply]
Glad I could help! Please don't forget to mark the answer. :slightly_smiling_face:

**Comment by Trout48** [comment]
Thank you, I will!
________________________________
